DataLayer/BatteryData/BatteryData.cpp

```diff
@@ -312,7 +309,7 @@ void BatteryData::setLowCellVoltageId(const unsigned char& lowCellVoltageId)
 
 void BatteryData::setHighCellVoltage(const unsigned short& highCellVoltage)
 {
-    highCellVoltage_ = (highCellVoltage / DECI_TO_ONES);
+    highCellVoltage_ = highCellVoltage;
```

Comment:
We'll have to keep the conversions for low & high cell voltage. CCS doesn't convert that atm.